.github/workflows/notify-dco-failure.js

```diff
@@ -1,7 +1,8 @@
 module.exports = async ({ context, github }) => {
   const { owner, repo } = context.repo;
   const { number: issue_number } = context.issue;
-  const { sha: ref, user } = context.payload.pull_request.head;
+  const ref = context.payload.pull_request.head.sha;
+  const { user } = context.payload.pull_request;
```

Comment:
Why not write `const user = context.payload.pull_request.user` which looks easier to read ?

Reply:
I think it depends. To me, `const { user } = context.payload.pull_request;` looks cleaner sicne you don't need to type `user` twice.

Reply:
`const user = context.payload.pull_request.user` vioaltes the `prefer-destructuring` rule of ESLint.

![image](https://user-images.githubusercontent.com/17039389/141949251-63178e45-39fb-4e5a-b205-f55e19787045.png)

https://eslint.org/docs/rules/prefer-destructuring